HelloWorld.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("hello world")
+}
```

Comment:
we also can display a message without import "fmt"
like:

package main

func main() {
   println("hello, world")
}